xunit-autolabeler-v2/ast_parser/python/source_parsers/test_data/direct_invocation_example.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
**N.B:** files in this `test_data` directory are only used as fixtures for testing. (They're not used in the core codebase at all.)